### PR TITLE
Add obs-virtualcam 1.0.0

### PR DESCRIPTION
**Pull request by claui**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/obs-virtualcam.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'obs-virtualcam' do
+  version '1.0.0,5db91b4'
+  sha256 'b71db5e3ecc9b64d79452f87bb43e2d0160e9189f90bad98a6e7b80f9fd5458e'
+
+  url "https://github.com/johnboiles/obs-mac-virtualcam/releases/download/v#{version.before_comma}/obs-mac-virtualcam-#{version.after_comma}-v#{version.before_comma}.pkg"
+  appcast 'https://github.com/johnboiles/obs-mac-virtualcam/releases.atom'
+  name 'OBS Virtual Camera'
+  homepage 'https://github.com/johnboiles/obs-mac-virtualcam'
+
+  # https://github.com/johnboiles/obs-mac-virtualcam/issues/85
```
Comment (vitorgalvao) [inline Casks/obs-virtualcam.rb:10]: Why do you feel the comment should be there?

Comment (claui) [reply Casks/obs-virtualcam.rb:10]: Just for convenience so we can quickly check whether the fix has been released yet.

**Comment by 0az** [comment]
> https://github.com/johnboiles/obs-mac-virtualcam/releases/tag/v1.1.0

As of yesterday's release of v1.1.0, there's support for High Sierra and Mojave.

File: Casks/obs-virtualcam.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/obs-virtualcam.rb:10]: > Shall we lose the comment?

I was going to say we could keep it, but turns out [we no longer need to](https://github.com/Homebrew/homebrew-cask/pull/82572#issuecomment-629483669).

**Comment by claui** [comment]
Thanks @0az for the heads up!
Bumped version.

**Pull request merged**